mne/inverse_sparse/tests/test_mxne_inverse.py

```diff
@@ -110,6 +110,13 @@ def test_mxne_inverse():
     assert_array_almost_equal(stc.times, evoked_l21.times, 5)
     assert_true(stc.vertices[1][0] in label.vertices)
 
+    fwd = forward.copy()
+    fwd['sol']['data'][:, np.random.randint(1000)] = 0.
+    stc, _ = mixed_norm(evoked_l21, fwd, cov, alpha, loose=None,
+                        depth=depth, maxit=300, tol=1e-8,
```

Comment:
maxit=3 and tol=np.inf
should be enough for testing purposes.
Also here say also why you do this. Same as above